### ISSUE TITLE
BG Honor GAIN

### ISSUE THREAD

**Issue by smlk87** [SunwellWoW/Sunwell-TBC-Bugtracker#3661]
Description: 
Fix Honor gain and all the pvp system .plz is all wrong.honor gain is too low 
when you lose you get no honor ,and when you win you dont take +honor ,tbc is not vanila.
ENd Fix the Qs too 

How it works: 

How it should work: 

Source (you should point out proofs of your report, please give us some source):

**Comment by baade123** [comment]
yeah this is pretty disapointing, even if its intended, the low population + Long waits for que's cannot handle such low rewards...

**Comment by kinghunterx** [comment]
bg queues are shit because the fucking honor gains are terrible so people have no motivating to queue bgs and to do pvp, fix this or add the bg marks exchange quest for honor otherwise welcome to a dead pvp server

**Comment by nerfedwow** [comment]
Mark turn in quest was put in during vanilla. Putting them in would fix the honor grind.

http://wowwiki.wikia.com/wiki/Patch_1.8.0
https://us.battle.net/forums/en/wow/topic/20765028576#post-5

-Each Battleground has an individual turn-in quest associated with its related victory/loss condition. Upon completion of a Battleground, the winning side will be awarded three Marks of Honor while the losing side will be awarded one Mark of Honor. Three Marks of Honor may be turned in to the associated questgiver for Honor points and faction increase. Should players complete this turn-in quest, they will find that the bonus Honor Points for completion are twice that of normal turn-in conditions (individual).

**Comment by Riztazz** [comment]
Yeah, we will add those, im also looking for information about honor gains for bg winning during tbc times, got any idea about those? as in for winning BG alone :p

**Comment by kinghunterx** [comment]
when will this be fixed @Riztazz ? pls implement honor gain asap otherwise pvp scene will be dead, this needs an urgent fix

**Comment by kinghunterx** [comment]
there is only 3 bgs popping everytime considering pvp was supposed to appeal on NW this is pretty worrying

**Comment by smlk87** [comment]
well atm more and more ppl being 70 the Qs will gonna be better 
but still we need AV .
and About the honor gain when you win or lose i am not sure if i am right but i think when you won you get somethink more + when you lose you get honor not like now.

**Comment by nerfedwow** [comment]
@Riztazz The battleground bonus honor weekends aren't working correctly. We should be getting double rep and honor during the weekend. The emissaries show up, but no bonus. They were added in patch 1.7.

http://wowwiki.wikia.com/wiki/Patch_1.7.0
Battleground "holidays" have been added to Warsong Gulch, Alterac Valley and Arathi Basin. Holidays occur during most weekends, starting on Thursday night at midnight and continuing until Tuesday morning. During a holiday, emissaries from that Battleground will be found in the major cities, and honor/faction rewards for performing objectives in that battleground are increased.

**Comment by nerfedwow** [comment]
Also in patch 2.1 they started forcing premades to go against other premades. It wasn't as big of an issue in retail as it is on private server because of the population. 

http://wowwiki.wikia.com/wiki/Patch_2.1.0
The new battleground matchmaking system is now implemented and active. This system allows the battleground to select teams of similar equipment quality and organizational level to battle each other. For now, the system will be very forgiving about creating matchups in order to keep queue times low. However, the parameters will be adjusted as necessary when more organized teams become active in the battlegrounds.

**Comment by nerfedwow** [comment]
Here is a post from 2006 with some numbers on the cost of items. including a screen shot of the prices
http://www.blue.cardplace.com/newcache/en/88829547.htm

http://oi3.tinypic.com/3zteee8.jpg
http://i11.tinypic.com/2r5rci1.jpg

-------------- level 70 Rewards ----- 
Prices on other versions appear to be as follows: http://i14.tinypic.com/3zteee8.jpg 

(Epic 70) Marshal's/General's Belt 14000 pts & 40 AB tokens 
(Epic 70) Marshal's/General's Wrist 9250 pts & 20 WSG tokens 
(Epic 70) Marshal's/General's Boots 14000 pts & 40 EotS tokens 

Sergeant's Cape/Cloak (epic 70) 9250 points & 20 AB tokens 

(Non-Epic 70) Gr& Marshal's/High Warlord's Shoulders 5500 pts & 20 AB 
(Non-Epic 70) Gr& Marshal's/High Warlord's Pants 8750 pts & 30 WSG 
(Non-Epic 70) Gr& Marshal's/High Warlord's Helm 8740 pts & 30 AV 
(Non-Epic 70) Gr& Marshal's/High Warlord's Gloves 5500 pts & 20 AV 
(Non-Epic 70) Gr& Marshal's/High Warlord's Chest 9000 pts & 30 AB 

You can see 70 blue weapons here http://i11.tinypic.com/2r5rci1.jpg 

Marshal's/General's 2h 25000 & 40 AV 
Marshal's/General's 1h 15000 & 20 EotS 


---------- Arena Rewards ----------- 
Helms are 1875 
Gloves are 1125 
Shoulders are 1500

Two-h& weaps (& bow/guns) are 3750 arena points 
One-h& weaps (& the shields/offh&s) are 1875 arena points

**Comment by smlk87** [comment]
and also i think you need to start focusing on the pvp issues not only the pve.inside on everyday updates .atm a couple of people care about it.and us you see is the the first post that have so many replays and comments .thats need  **High Priority.**

**Comment by Riztazz** [comment]
Aight, finally some legit info, getting into it then

edit: Honor will be doubled for specific bg that is current holiday BG and i will implement mark of honor quests for specific BGs

**Comment by smlk87** [comment]
Awsome! thanks we are wating
